operators/multiclusterobservability/manifests/base/alertmanager/alertmanager-statefulset.yaml

```diff
@@ -138,6 +146,36 @@ spec:
           readOnly: true
         - mountPath: /etc/proxy/secrets
           name: alertmanager-proxy
+      - image: quay.io/brancz/kube-rbac-proxy:v0.15.0
```

Comment:
We can't rely on this image because of compliance. Need to use `quay.io/stolostron/kube-rbac-proxy`